packages/api-client/src/executions.ts

```diff
@@ -208,9 +211,10 @@ export const searchExecutionsByGranules = async (params: {
         'Content-Type': 'application/json',
       },
       path: '/executions/search-by-granules',
+      queryStringParameters: query,
```

Comment:
queryStringParamaters needed to be added as the default logic for this endpoint, even with granules specified, is to use a limit of `1` for the RDS query.  

Also there's no good reason we shouldn't be able to specify the `query` object here.